### DISCUSSION TITLE
I keep getting this error only on linux

### DISCUSSION THREAD

**Discussion by xgui4** [pyinstaller/pyinstaller#7737] [Help]
xgui4@xgui4-VirtualBox:~/Téléchargements/Windows-95---The-shooter-game--master/d
ist$ '/home/xgui4/Téléchargements/Windows-95---The-shooter-game--master/dist/GlassOS95' 
pygame 2.4.0 (SDL 2.26.4, Python 3.10.6)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "GlassOS95.py", line 14, in <module>
FileNotFoundError: No file 'assets/Background.png' found in working directory '/home/xgui4/Téléchargements/Windows-95---The-shooter-game--master/dist'.
[2336] Failed to execute script 'GlassOS95' due to unhandled exception!

and yes i try to add data but with no prevail at all

**Comment by rokm** [comment]
>  FileNotFoundError: No file 'assets/Background.png' found in working directory '/home/xgui4/Téléchargements/Windows-95---The-shooter-game--master/dist'.

Do not reference to files using relative paths, which anchors them to the current working directory, which in turn depends on where you launch your program from. Use absolute paths, anchored to `__file__`, like suggested by [documentation](https://pyinstaller.org/en/stable/runtime-information.html#using-file).
